docs/bootstrap.md

```diff
@@ -0,0 +1,19 @@
+# Being Strict
+
+Start being strict is very easy, at first we just need to configure an instance of `\StrictPhp\StrictPhpKernel`
```

Comment:
To bootstrap StrictPhp, we initialize the `StrictPhp\StrictPhpKernel` singleton: